### ISSUE TITLE
Adding Dropout during Training 

### ISSUE THREAD

**Issue by kabbas570** [qubvel/segmentation_models.pytorch#708]
Hello, thanks for the nice library,
Is there any way to add drop out during training to avoid overfitting during training?

Thanks

**Comment by jerofad** [comment]
Do you have a solution to this?. I want to avoid implementing if from scratch for all the architectures I want to use.

**Comment by rabinadk1** [comment]
I am seeking a solution to this too.

**Comment by qubvel** [comment]
Many encoder architectures by default have some regularization such as dropout, drop connect, or drop path.
That is why there is no such option to add any additional dropout layers.
